docs/pull_request_template.md

```diff
@@ -0,0 +1,21 @@
+
+
+:zap: Please ensure that you reviewed the [guidelines for contributing](/CONTRIBUTING.md) to this repository. 
+
+
+What does this pull request contain? Please explain your changes.
+---------------------------------------------------
+* ...
+
+* ...
+
+* ...
+
+Does this close any currently open issue(s)?
+------------------------------------------
+…
+
+
+Any other comments?
+-------------------
+…
```

Comment:
Does this create new issues or are any other issues or pull requests affected by this?